sample-apps/react-native/dogfood/ios/StreamReactNativeVideoSDKSample/Info.plist

```diff
@@ -68,12 +68,6 @@
 	<string>$(PRODUCT_NAME) requires camera access in order to capture and transmit video</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>$(PRODUCT_NAME) requires microphone access in order to capture and transmit audio</string>
-	<key>NSPhotoLibraryAddUsageDescription</key>
-	<string>$(PRODUCT_NAME) requires access to photo library to allow you to share photo
-			attachments</string>
-	<key>NSPhotoLibraryUsageDescription</key>
-	<string>$(PRODUCT_NAME) requires access to photo library to allow you to share photo
-			attachments</string>
 	<key>RTCAppGroupIdentifier</key>
```

Comment:
Removing both is wrong,

`NSPhotoLibraryUsageDescription` is still required for native image picker